components/common/map/LocationSearch.tsx

```
@@ -0,0 +1,185 @@
+      return;
+    }
+
+    setIsLoading(true);
+    try {
+      const response = await fetch(
+        `https://nominatim.openstreetmap.org/search?format=json&q=${encodeURIComponent(query)}&limit=5`
+      );
+      const data = await response.json();
+      setSearchResults(data);
+    } catch (error) {
+      console.error('Error searching location:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)